harness/determined/cli/trial.py

```diff
@@ -121,6 +126,77 @@ def trial_logs(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def generate_support_bundle(args: Namespace):
+    temp_dir = tempfile.TemporaryDirectory()
+    trial_logs_filepath = write_trial_logs(args, temp_dir)
+
+    master_logs_filepath = write_master_logs(args, temp_dir)
+
+    api_experiment_filepath, api_trail_filepath = write_api_call(args, temp_dir)
+
+    bundle = None
+    if args.output_dir is None:
+        output_dir = os.getcwd()
+
+    else:
+        bundle = tarfile.open(f"{args.output_dir}.tar.gz", "w:gz")
```

Comment:
something isn't quite right here - why are we opening two tar files? did we just mean to set `output_dir = args.output_dir` here instead? `--output-dir` doesn't work right now, if I try to use it I see this because we never set output dir. 
```
(det) ➜  determined git:(b46d5721a) ✗ det t support-bundle --output-dir tmp 4   
Traceback (most recent call last):
  File "/Users/bradleylaney/Code/determined/harness/determined/cli/cli.py", line 247, in main
    parsed_args.func(parsed_args)
  File "/Users/bradleylaney/Code/determined/harness/determined/common/api/authentication.py", line 397, in f
    return func(namespace)
  File "/Users/bradleylaney/Code/determined/harness/determined/cli/trial.py", line 149, in generate_support_bundle
    fullpath = os.path.join(output_dir, tar_filename)
UnboundLocalError: local variable 'output_dir' referenced before assignment
Failed to support bundle
```

Reply:
Yeah, I put a review somewhere pointing this out but I think it might have gotten lost somewhere...

Reply:
Yes, I think I didn't commit this change correctly, I fixed it now!